textfinding_indikator1.py

```diff
@@ -0,0 +1,120 @@
+# this code is for TEXT FINDING - INDIKATOR 1
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+    init_keywords = list(keyword)
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            if re.search(keyword[0], line, re.IGNORECASE):
+                result.append([idx, line])
+
+        elif (lv == 2):
+            # cek dgn keyword 'domain'. cont: domain arsitektur proses bisnis, domain data dan informasi, domain arsitektur keamanan, dll
+            for key in init_keyword:
+                if key in keyword:
+
+                    # manfaatin regex untuk searching yg not only kata keyword
+                    reg = f'(?:(domain)\s+(arsitektur)?\s?({key}))'
+                    if re.search(reg, line, re.IGNORECASE):
+                        result.append([idx, line])
+                        keyword.remove(key)
+
+        elif (lv == 3):
+            # masih pengecekan keyword yg sama dgn level 2, tapi tanpa 'domain'
+            for key in init_keyword:
+                if key in keyword:
+
+                    # gausah regex krn searching nya langsung kata keyword tanpa ada embel embel 'domain'
+                    if re.search(key, line, re.IGNORECASE):
+                        result.append([idx, line])
+
+                        # hapus key yg udh ditemuin supaya gausah di search lagi
+                        keyword.remove(key)
+
+        else:  # lv == 4
+            # cek dulu keyword utama level 4 which is arsitektur spbe nasional
+            reg = f'(?:(arsitektur)/s+(spbe)\s+(nasional))'
```

Comment:
I am just curious, is the forward slash `/` between the words arsitektur and spbe intended?